nixos/modules/services/networking/ntopng.nix

```diff
@@ -95,7 +95,7 @@ in
   config = mkIf cfg.enable {
 
     # ntopng uses redis for data storage
-    services.redis.enable = true;
+    services.redis.servers."".enable = true;
```

Comment:
Shouldn't the instance have a name?

Reply:
Uh, you are right -- forgot to set it when testing.